### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by binodbhusal**
**The following activities carried out in this pull request**
- Added event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implemented a function for the "Clear all completed" button.

### CODE REVIEW

File: modules/functions.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    description: taskDescription,
+    completed: false,
+    index: tasks.length + 1,
+  };
+  getStorage(tasks);
+  tasks.push(newTask);
+  setStorage(tasks);
+  taskPopulate(tasks);
+  taskEntry.value = '';
+};
```
Comment (Zuhaib042) [inline modules/functions.js:104]: - [ ] You have completed all the requirements but there is a bug in your project. whenever I add the todos, they get stored in local storage but when I refresh the page, I can still see them in local storage but after refreshing the page, if I interact with the todos in any way, the list in the local storage gets updated and previous todos get removed.
kindly consider fixing this bug.

https://user-images.githubusercontent.com/96683843/236521513-d3483f55-dd39-496d-9268-ad31d37b3dbb.mov

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @binodbhusal 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Descriptive PR
✅ Correct Github flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]
Hi @binodbhusal 

Your project is complete!  There is nothing else to say other than... it's time to merge it :squirrel:

Congratulations! 

allfather-approves-approved

## Highlights

✔️ Good commit history
✔️ Good work implementing changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Review by BregornOriginal — APPROVED** [review]
Hi @binodbhusal 

Your project is complete!  There is nothing else to say other than... it's time to merge it :squirrel:

Congratulations! 

allfather-approves-approved

## Highlights

✔️ Good commit history
✔️ Good work implementing changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Review by BregornOriginal — APPROVED** [review]
Hi @binodbhusal 

Your project is complete!  There is nothing else to say other than... it's time to merge it :squirrel:

Congratulations! 

allfather-approves-approved

## Highlights

✔️ Good commit history
✔️ Good work implementing changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**